features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1195,6 +1195,15 @@
         {% if oauth.extensions.scope_metadata_service is defined %}
         <ScopeMetadataService>{{oauth.extensions.scope_metadata_service}}</ScopeMetadataService>
         {% endif %}
+
+        {% if oauth.restricted_query_parameters is defined %}
+        <!-- Configuration to pass a list of keys which are restricted to be sent as query parameters. -->
+        <RestrictedQueryParameters>
+            {%for query_parameter in oauth.restricted_query_parameters%}
+                <RestrictedQueryParameter>{{query_parameter.key}}</RestrictedQueryParameter>
```

Comment:
`RestrictedQueryParameter` this is redundant since this would be included in the `RestrictedQueryParameters` tag. something like `Parameter` would suffice. wdyt?

Reply:
Fixed with 065498b5713a723c255384aa986ce08c3c5ca203